cmd/juju/application/deployer/bundle.go

```diff
@@ -94,6 +94,9 @@ func (d *deployBundle) deploy(
 		return errors.Annotatef(err, "cannot deploy bundle")
 	}
 	d.bundleDir = d.bundleDataSource.BasePath()
+	if bundleData.UnmarshaledWithServices() {
+		logger.Warningf(`"services" key found in bundle file is deprecated, superseded by "applications" key.`)
```

Comment:
This is the actual change.

Reply:
I think we should also point out that `services` entries will be ignored or something along those lines. This reads like a deprecation warning and users might ignore the change log and think that it still works as expected.

A perhaps better (albeit a bit more brute force) way would be to simply return an error and abort the deploy steps altogether.

Reply:
Apparently, I got confused by the upcoming changes for develop where we stop parsing `services`. Nevermind my comment above